### DISCUSSION TITLE
Question 51: Is verbatim locality information sufficient justification to create a unique site record? [IZ - KJ, RDLP, JD]

### DISCUSSION THREAD

**Discussion by fmjjones** [fieldmuseum/EMu-Documentation#817] [Q&A] [answered]
Description: Example site IRN: 731489. Is it acceptable to have multiple sites with verbatim locality information but not precise locality? If yes, should Rank = precise locality with an empty precise locality field?

**Comment by fmjjones** [comment]
Response: Revisit in Sites meeting

- Agenda is here: https://github.com/fieldmuseum/EMu-Documentation/discussions/811

**Comment by rondlg** [comment, marked as answer]
Yes, it’s fine to do that. Rank is Precise Locality.
- A blank Precise Locality indicates that more information eg lat lon may be found at a later time.
